conversion.py

```diff
@@ -1258,7 +1279,7 @@ def define(value, name, encode, decode):
 
         # 255 is reserved
         define(254, u"dispersy-missing-sequence", self._encode_missing_sequence, self._decode_missing_sequence)
-        define(253, u"dispersy-missing-proof", self._encode_missing_proof, self._decode_missing_proof)
+        define(253, u"dispersy-missing-proof", self._encode_missing_proof, self._decode_missing_proof) # self_decode_missing_proof returns a deferred.
```

Comment:
Not only `self_decode_missing_proof` returns a deferred now if I'm correct, also the other methods, so maybe you can add a comment above these defines that these decode methods are returning a deferred.